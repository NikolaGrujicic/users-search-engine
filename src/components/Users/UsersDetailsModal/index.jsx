import UsersDetailsModal from './UsersDetailsModal';

export default UsersDetailsModal;
