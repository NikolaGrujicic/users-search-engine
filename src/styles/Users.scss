* {
    box-sizing: border-box;
  }
  
  .card {
    margin: 5px;
    width: 300px;
    height: 300px;
  }
  
  .card img {
    width: 100px;
    height: 100%;
  }

  .card-footer {
      text-align: center;
  }
  
  .card > .card-body {
    height: 100%;
    overflow: auto;
    text-align: center;
    padding: 5px;
  }

  #users-container {
    margin-top: 100px;
    max-width: 1240px;
}