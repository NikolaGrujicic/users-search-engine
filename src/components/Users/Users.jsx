/* eslint-disable arrow-body-style */
import React from 'react';
import UsersDetailsModal from './UsersDetailsModal';

const Customers = () => {
  return (
    <div>
      <div id="users-container" className="container">
        <div className="row">
          <div className="card">
            <div className="card-body ">
              <div className="card-footer">
                <p className="card-text text-center text-capitalize text-primary">Nikola</p>
                <p className="card-text text-center text-capitalize text-primary">Grujicic</p>
              </div>
              <br />
            </div>
          </div>
        </div>
      </div>
      <UsersDetailsModal />
    </div>
  );
};

export default Customers;
