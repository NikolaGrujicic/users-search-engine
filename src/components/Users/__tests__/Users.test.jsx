import React from 'react';
import { shallow } from 'enzyme';
import Users from '../Users';

test('renders Users', () => {
  const wrapper = shallow(
    <Users />,
  );
  expect(wrapper).toMatchSnapshot();
});
