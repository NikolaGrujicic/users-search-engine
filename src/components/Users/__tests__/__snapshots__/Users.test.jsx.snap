// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Users 1`] = `
<div>
  <div
    className="container"
    id="users-container"
  >
    <div
      className="row"
    >
      <div
        className="card"
      >
        <div
          className="card-body "
        >
          <div
            className="card-footer"
          >
            <p
              className="card-text text-center text-capitalize text-primary"
            >
              Nikola
            </p>
            <p
              className="card-text text-center text-capitalize text-primary"
            >
              Grujicic
            </p>
          </div>
          <br />
        </div>
      </div>
    </div>
  </div>
  <UsersDetailsModal />
</div>
`;
